Jenkinsfile

```diff
@@ -137,16 +137,16 @@ spec:
                     ssh $sshHost rm -rf $deployParentDir/$GIT_BRANCH/$LATEST_DIR
                     ssh $sshHost mkdir -p $deployParentDir/$GIT_BRANCH/$LATEST_DIR
 
-                    cp $OUTPUT_THEIA_NAME-*.vsix $OUTPUT_THEIA_NAME.vsix
-                    scp $OUTPUT_THEIA_NAME.vsix $sshHost:$deployParentDir/$GIT_BRANCH/$LATEST_DIR/$OUTPUT_THEIA_NAME.vsix
+                    cp $OUTPUT_CHE_NAME-*.vsix $OUTPUT_CHE_NAME.vsix
```

Comment:
Is this going to require a change to the dev file used by codewind in che?

Reply:
yes https://github.com/eclipse/codewind-che-plugin/pull/152